packages/shared/src/components/banners/common.ts

```diff
@@ -1,6 +1,15 @@
 import classed from '../../lib/classed';
 
 export const bottomBannerBaseClasses =
-  'fixed bottom-0 left-0 z-modal flex w-full flex-row justify-center';
+  'fixed bottom-0 left-0 z-modal flex w-full flex-row justify-center border-none';
 
 export const BottomBannerContainer = classed('div', bottomBannerBaseClasses);
+
+export const authBannerBg = {
+  backgroundImage:
+    "url('https://daily-now-res.cloudinary.com/image/upload/s--lf8LUJjq--/f_auto/v1732012913/login-popover-dailydev_mxb7lw')",
+  backgroundSize: 'cover',
+};
```

Comment:
Maybe is better to use image (position absolute + low z-index) for this, enabling lazy loading and srcset (this image is HUGE)

Reply:
@ilasw I don't know if I'm getting a little confused here or not, but 9.9kb is surely not huge for an image that spans the width of the screen?

Reply:
This image is ~20kb before compression and can be ~10kb [after resizing at 1920px width](https://res.cloudinary.com/daily-now/image/upload/s--N2EPbaJl--/c_fit,w_1920/login-popover-dailydev_mxb7lw) (this is a good size for a good part of monitors). 

<img width="392" alt="image" src="https://github.com/user-attachments/assets/736a97d7-1cad-4aad-8248-c51ac24a9b95">

I think that is important to try have the smallest impact on page load if we can do this with low effort. Also we already gained lot of LCP using this hack in onboarding, so this is not a new pattern. wdyt?

Reply:
I'm a little confused. Can you explain where you are getting the 20KB number from? 
It's 10kb for me when looking in element, network tab and when downloading it

Styles:
![Screenshot 2024-11-20 at 16 40 31](https://github.com/user-attachments/assets/aeb1be4e-ae88-4f23-b045-f002e5cd2a63)

Network:
![Screenshot 2024-11-20 at 16 38 05](https://github.com/user-attachments/assets/b12f5d23-7662-40e5-9fdd-32471037ee08)

Download:
![Screenshot 2024-11-20 at 16 38 52](https://github.com/user-attachments/assets/48ed6f13-3ebc-4d4a-bcd0-ca0f354f1af0)

Reply:
This is the result [using the cropped image](https://res.cloudinary.com/daily-now/image/upload/s--qPSnoHHa--/c_auto,g_center,w_1920/f_auto/login-popover-dailydev_mxb7lw). Your call if implement the srcset but I strongly suggest to use it

<img width="699" alt="image" src="https://github.com/user-attachments/assets/5c309e1e-42f8-4b1b-84b8-799a3b4a9f5d">

Reply:
@ilasw  Added it.

Thanks for uploading the sizes to cloudinary